api/src/main/java/com/alibaba/nacos/api/naming/utils/NamingUtils.java

```diff
@@ -73,4 +86,21 @@ public static void checkServiceNameFormat(String combineServiceName) {
                     "Param 'serviceName' is illegal, it should be format as 'groupName@@serviceName'");
         }
     }
+    
+    /**
+     * Returns a combined string with serviceName and groupName. Such as 'groupName@@serviceName'
+     * <p>This method works similar with {@link com.alibaba.nacos.api.naming.utils.NamingUtils#getGroupedName} But not verify any parameters.
+     *
+     * </p> etc:
+     * <p>serviceName | groupName | result</p>
+     * <p>serviceA    | groupA    | groupA@@serviceA</p>
+     * <p>nil         | groupA    | groupA@@</p>
+     * <p>nil         | nil       | @@</p>
+     *
+     * @return 'groupName@@serviceName'
+     */
+    public static String getGroupedNameOptional(final String serviceName, final String groupName) {
+        final String resultGroupedName = groupName + Constants.SERVICE_INFO_SPLITER + serviceName;
+        return resultGroupedName.intern();
```

Comment:
If it is not a fixed variable, do not do this manually to get it into the constant pool

Reply:
Already update